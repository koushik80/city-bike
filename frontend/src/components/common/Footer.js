import React from 'react';
import { Container, Grid } from '@mui/material';
import { Link } from 'react-router-dom';
import './Footer.scss';

const Footer = () => {
    return (
        <footer className='footer'>
            <Container>
                <Grid container spacing={2}>
                    <Grid item xs={12} md={6} lg={3}>
                        <ul>
                            <li>
                                <Link to="/">Link 1</Link>
                            </li>
                            <li>
                                <Link to="/">Link 2</Link>
                            </li>
                            <li>
                                <Link to="/">Link 3</Link>
                            </li>
                        </ul>
                    </Grid>
                    <Grid item xs={12} md={6} lg={3}>
                        <ul>
                            <li>
                                <Link to="/">Link 1</Link>
                            </li>
                            <li>
                                <Link to="/">Link 2</Link>
                            </li>
                            <li>
                                <Link to="/">Link 3</Link>
                            </li>
                        </ul>
                    </Grid>
                    <Grid item xs={12} md={6} lg={3}>
                        <ul>
                            <li>
                                <Link to="/">Link 1</Link>
                            </li>
                            <li>
                                <Link to="/">Link 2</Link>
                            </li>
                            <li>
                                <Link to="/">Link 3</Link>
                            </li>
                        </ul>
                    </Grid>
                </Grid>
            </Container>
            <div className="footer--copy-right-text">
                <b>All rights reserved © koushik@devHuß {new Date().getFullYear()}</b>
            </div>
        </footer>
    );
};

export default Footer;