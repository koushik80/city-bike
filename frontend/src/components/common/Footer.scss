.footer {
    margin-top: 2rem;
    background-color: #5670ff;
    padding: 2rem;
    padding-bottom: 4.5rem;
    bottom: 0;

    ul {
        text-align: center;

       a {
        color: white;
       }
    }

    &--copy-right-text {
        font-size: 12px;
        text-align: center;
        margin-top: 2rem;
        color: white;
    }
}