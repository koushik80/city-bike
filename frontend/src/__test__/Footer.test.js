import React from 'react'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Footer from '../components/common/Footer'


describe('Footer', () => {

  it('should render correctly', () => {
    const wrapper = renderer.create(<BrowserRouter><Footer /></BrowserRouter> )

    expect(wrapper).toMatchSnapshot()
  })
})