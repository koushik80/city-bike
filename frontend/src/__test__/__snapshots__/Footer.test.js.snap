// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 MuiGrid-grid-lg-3 css-72mr71-MuiGrid-root"
      >
        <ul>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Pricing
            </a>
          </li>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Blog
            </a>
          </li>
        </ul>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 MuiGrid-grid-lg-3 css-72mr71-MuiGrid-root"
      >
        <ul>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              FAQ
            </a>
          </li>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Help
            </a>
          </li>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Feedback
            </a>
          </li>
        </ul>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 MuiGrid-grid-lg-3 css-72mr71-MuiGrid-root"
      >
        <ul>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Terms
            </a>
          </li>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Privacy
            </a>
          </li>
          <li>
            <a
              href="/contact"
              onClick={[Function]}
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="footer--copy-right-text"
  >
    <b>
      All rights reserved © koushik@devHuß 
      2022
    </b>
  </div>
</footer>
`;
